server/views/pages/functionalSkills/index.njk

```diff
@@ -28,61 +28,114 @@
 
   {% include "../../partials/prisonerBanner.njk" %}
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <span class="govuk-caption-l">Learning and work progress</span>
-      <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+  {% if not allFunctionalSkills.problemRetrievingData %}
 
-      <div class="govuk-summary-card">
-        <div class="govuk-summary-card__title-wrapper">
-          <h2 class="govuk-summary-card__title">Current functional skills</h2>
-          {# Not MVP, Add assesment link
-          <ul class="govuk-summary-card__actions">
-              <li class="govuk-summary-card__action">
-                <a class="govuk-link" href="">Update</a>
-              </li>
-          </ul>
-          #}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l">Learning and work progress</span>
+        <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+
+        <div class="govuk-summary-card">
```

Comment:
This is the table from the earlier PR, where it displays the latest functional skills - its just been indented because the whole lot is within the new if block